quipucords/scanner/network/runner/roles/jboss_ws/tasks/main.yml

```diff
@@ -21,7 +21,7 @@
 
 # If client has not renamed default JWS_HOME folder and followed recommended zip file install instructions, JWS_HOME should be in /opt.
 - name: search for JWS_HOME in search directories
-  raw: export LANG=C LC_ALL=C; find {{search_directories}} -type d 2>/dev/null | egrep '*(ews|jws)*[0-99]\.[0-99]$'
+  raw: export LANG=C LC_ALL=C; find {{search_directories}} -type d 2>/dev/null | egrep '(ews|jws).*[0-99]\.[0-99]$'
```

Comment:
Someone clearly confused wildcard with regexp here. The previous version was "ews or jws, zero or more times", and it matched anything. This resulted in considering things like `/usr/lib64/python3.6`, `/usr/lib/firmware/ath10k/WCN3990/hw1.0` or even `/sys/devices/pci0000:00/0000:00:01.2`, which clearly do not contain JWS.

Paired with [DISCOVERY-544](https://issues.redhat.com/browse/DISCOVERY-544), it made extended search actually worse at detecting JWS than standard search.